src/cli/commands/force/lightning/local/__tests__/setup.test.ts

```diff
@@ -1,112 +0,0 @@
-/*
```

Comment:
All of the tests files where moved to a different path but for some reason, during diff, they are being shown as deleted + new ones added which makes code reviewing more difficult 😞